standards/problem-solving.md

```diff
@@ -0,0 +1,24 @@
+# A Useful Guide To Problem solving
+Problem solving is a fundamental key to development. As developers, we are given tasks to complete on a project. The idea of trying to figure out the issue all at once is recipe for disaster. Developers could potentially face imposter syndrome, burnout, or the task being delayed due to the lack of problem solving skills. It does not take a genius to problem solve but a developer who is dedicated to learn problem solving techniques.
+It is important to remember that problem solving is not a one size fit all solution. We all have a mind of our own so to expect every developer to follow the same approach is not realistic. However, developers will be provided with different problem techniques in this page to help them become better at problem solving. 
+
+## The Breakdown 
+A developer have been tasked to a new issue by their lead. The initial thought maybe, "how in the hell will I figure this out"? While being a dev, you will be asked to do find solutions to problems developers have never solved before. Instead of allowing the unknown to overwhelm the dev, try breaking down the problem using **psuedocode**.
+
+- Psuedocode is used to break down problems by using steps to find the solution. It does not require actual syntax to be written out, and it is human readable which means it does not get compiled by the computer. developers can write the steps on notepad, whiteboard, or even make temporary comments in a IDE. The point is to breakdown a huge problem into chucks and connect the dots to find a solution in an efficient way. Once the steps seem logical, developers will attempt to convert this psuedocode into code and test their logic. Read more about this here [What is Pseudocode? How to Use Pseudocode to Solve Coding Problems](https://www.freecodecamp.org/news/what-is-pseudocode-in-programming/)
+
+
+## Find Resources 
+It is impossible to retain every thing learned while being a developer. Google is your best friend. The amount of resources available to help complete task is vast. For Example, [MDN](https://developer.mozilla.org/en-US/), [CSS-Tricks](https://css-tricks.com/), [Javascript.info](https://javascript.info/), are a few resources for front-end development. Reading Documentation on how a certain library work is key when using it on a project. The documentation should always be a reference when using it. Search for your problem using the error message keywords given to you. Other developers have probably ran into the same error as you and may have solved it. How you google is just as important as what you google, so click here to learn. [11 Tricks To Master the Art of Googling as a Software Developer](https://betterprogramming.pub/11-tricks-to-master-the-art-of-googling-as-a-software-developer-2e00b7568b7d)  
```

Comment:
How do you feel about changing `every thing` to `everything`? https://writingexplained.org/everything-or-every-thing-difference looks like the biggest difference is the addition of adverbs or verbs in between the every and thing on the first spelling, which is not the case above.

Reply:
I am cool with that